### DISCUSSION TITLE
Manually point to camera model in the .db

### DISCUSSION THREAD

**Discussion by Rarscal** [alicevision/Meshroom#2217] [Q&A] [answered]
Hello, currently having an issue which pops up around the meshing stage of the process.
I've learned this is due to the images having no assigned camera model.

I searched through the .db preparing to add the details, but realised it already has my camera model (Fujifilm X-T20) but even with the files I'm importing tagged as such in the Metadata/EXIF info, it still refuses to recognise my camera brand and model.

I've searched online and can't seem to find a way to manually point Meshroom to what the camera is, it seems it entirely relies on this automated process and if it fails you're sort of out of luck?

I may be missing something, but it's quite odd that you, knowing fully well what you used to shoot the footage on, can't tell the software exactly what it is.
Any help is appreciated, lest I need to download Blender to do the geometry gen/Camera solve.

Cheers

**Comment by natowi** [comment, marked as answer]
```
[warning] The camera found in the sensor database is slightly different for image(s):
[warning] image: '7.jpg'
        - image camera brand: FUJIFILM
        - image camera model: X-T20
        
        - sensor database camera brand: Fujifilm
        - sensor database camera model: Fujifilm X-T20
        - sensor database camera sensor width: 23.6 mm
```

So change the db entry to FUJIFILM;X-T20;23.6;

Also if your image is named like 12345676543234.jpg there may be a error message about "corrupted image". https://github.com/alicevision/AliceVision/issues/1411

Tested with the image from https://www.dpreview.com/sample-galleries/7251932144/fujifilm-x-t20-sample-gallery/7318731257 (image name had to be shortened, but image was recognized fine.)

And yes, there is a way to manually input the values in the camerainit node:
![grafik](https://github.com/alicevision/Meshroom/assets/8138537/94738e9d-2b3c-4caf-a215-7ad377ed5191)



There are also other options for the camera information like camera calibration or the LCP info, but the options mentioned above should suffice.
